### PR TITLE
lambda - introducing image_uri

### PR DESCRIPTION
**Pull request by manuelschneider**
Hi, please see #1417 for how and why :) Thanks! ~Manuel

### CODE REVIEW

File: plugins/modules/lambda.py [hunk 1]
```diff
@@ -22,6 +22,12 @@
     default: present
     choices: [ 'present', 'absent' ]
     type: str
+  image_uri:
+    description:
+      - The Amazon ECR URI of the image to use.
+      - Required (alternative to runtime zip_file and s3_bucket) when creating a function.
+      - Required when I(state=present).
+    type: str
```
Comment (alinabuzachis) [inline plugins/modules/lambda.py:30]: Please add `version_added: 6.0.0`

**Review by alinabuzachis — COMMENTED** [review]
Thank you @manuelschneider. Can you please add a changelog fragment https://docs.ansible.com/ansible/latest/community/development_process.html#id11 and some integration tests for this new parameter?

**Comment by nethershaw** [comment]
@manuelschneider Do you intend to acknowledge the request made of you eight months ago, mere days after you opened this PR?

**Comment by tremble** [comment]
closing in favour of #1939

**Pull request closed without merging**